src/pages/SettingsPage/UsersSettings/DeleteUserDialog.tsx

```diff
@@ -12,14 +12,14 @@ type DeleteUserDialogProps = {
 const DeleteUserDialog = ({ onHide, selectedUsers, onDelete, onDisable }: DeleteUserDialogProps) => {
   const [value, setValue] = useState('')
 
-
   if (!selectedUsers?.length) {
     // this shouldn't happen
     onHide()
     return <></>
   }
 
   const selectedUsersString = selectedUsers.join(', ')
+  const confirmDeleteUsersString = selectedUsers?.length > 1 ? "delete selected" : selectedUsersString
```

Comment:
Since `selectedUsers?.length` may be `null` (I assume?) this `> 1` check I suppose wouldn't work. Not sure what the best javascript equivalent is.

However, since above we're returning early on `!selectedUsers?.length` already I guess we're safe?

Reply:
This may then be clearer that we're not expecting `null` selectedUsers to begin with.
```suggestion
  const confirmDeleteUsersString = selectedUsers.length > 1 ? "delete selected" : selectedUsersString
```